recipe/meta.yaml

```diff
@@ -74,7 +75,7 @@ outputs:
         - numpy
         - scipy
           # mkl variant pulls in intel-openmp which conflicts with llvm-openmp. i.e. force to use openblas variant of numpy, scipy, etc.
-        - blas=*=openblas  # [osx and x86_64]
+        - blas =*=openblas  # [osx and x86_64]
```

Comment:
is this intended?

Reply:
`version_constraints_missing_whitespace` see https://github.com/AnacondaRecipes/xgboost-feedstock/pull/20#issuecomment-2286119622

Reply:
oh I see, thanks